spring-cloud-kubernetes-commons/src/main/java/org/springframework/cloud/kubernetes/commons/config/ConfigMapConfigProperties.java

```diff
@@ -71,11 +71,10 @@ public void setSources(List<Source> sources) {
 	 */
 	public List<NormalizedSource> determineSources() {
 		if (this.sources.isEmpty()) {
```

Comment:
no need to reference via `ConfigMapConfigProperties.this`, those properties are inherited anyway.